tools/vp20compiler/main.c

```diff
@@ -499,7 +504,9 @@ void translate(const char* str)
             struct prog_src_register reg = ins.SrcReg[j];
             if (reg.File != PROGRAM_UNDEFINED) {
                 // printf(" - in %d\n", j);
-                assert(!reg.RelAddr); //TODO: A0 rel
+                if(reg.RelAddr) {
+                    vsh_set_field(vsh_ins, FLD_A0X, 1);
```

Comment:
Shouldn't this only work for `PROGRAM_ENV_PARAM` and be forbidden otherwise?

Also, to what source-reg does it apply? - shouldn't `j` be checked?
(Same goes for `FLD_CONST` which already existed - do we already prevent code from using it in multiple src registers?)